storage_v2/factoryadapter/reader.go

```diff
@@ -49,7 +49,7 @@ func (tr *TraceReader) GetOperations(ctx context.Context, query tracestore.Opera
 		ServiceName: query.ServiceName,
 		SpanKind:    query.SpanKind,
 	})
-	var operations []tracestore.Operation
+	operations := []tracestore.Operation{}
```

Comment:
@yurishkuro I patched this to initialize an empty slice so we don't return an empty slice as nil

Reply:
the original version was correct. If there is an error, it is expected that we return nil.

Reply:
@yurishkuro but if there's no error and it is an empty list then we would also return nil - should we do an early return ?

Reply:
ok